src/index.html

```diff
@@ -7,6 +7,218 @@
     <link rel="stylesheet" href="./styles/main.css">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#">
+        <img class="header__logo" src="images/logo.png" alt="moyo logo">
+      </a>
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__list_item">
+            <a class="nav__list_link is-active" href="#">Apple</a>
+          </li>
+          <li class="nav__list_item">
+            <a class="nav__list_link" href="#">Samsung</a>
+          </li>
+          <li class="nav__list_item">
+            <a class="nav__list_link" href="#">Смартфоны&nbsp;и&nbsp;телефоны</a>
+          </li>
+          <li class="nav__list_item">
+            <a class="nav__list_link" href="#" data-qa="nav-hover">Ноутбуки&nbsp;и&nbsp;компьютеры</a></li>
+          <li class="nav__list_item">
+            <a class="nav__list_link" href="#">Гаджеты</a>
+          </li>
+          <li class="nav__list_item">
+            <a class="nav__list_link" href="#">Планшеты</a>
+          </li>
+          <li class="nav__list_item">
+            <a class="nav__list_link" href="#">Фото</a>
+          </li>
+          <li class="nav__list_item">
+            <a class="nav__list_link" href="#">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="catalog">
+        <div class="card" data-qa="card">
+          <img src="images/imac.jpeg" alt="" class="card__image">
+          <span class="card__name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</span>
+          <span class="card__id">Код товара: 195434</span>
+          <div class="card__reviews review-box">
+              <div class="review-box__stars stars__4">
```

Comment:
этот блок следует называть так, чтобы переиспользовать, не привязывая к карточке
```suggestion
              <div class="stars stars__4">
```
количество звезд - модификатор
```suggestion
              <div class="stars stars--4">
```